web/src/app/admin/indexing/status/page.tsx

```diff
@@ -67,7 +74,7 @@ function Main() {
   } = useSWR<ConnectorIndexingStatus<any>[]>(
     "/api/manage/admin/connector/indexing-status",
     fetcher,
-    { refreshInterval: 30000 } // 30 seconds
+    { refreshInterval: 30000 }, // 30 seconds
```

Comment:
Someone noticed that the prettier we use is actually:
npx prettier --write --trailing-comma es5 .

please run with --trailing-comma es5